docs/tutorials/09_ryberg_dynamics.ipynb

```diff
@@ -0,0 +1,1088 @@
+      "text/plain": [
+       "<Figure size 496.776x284.278 with 1 Axes>"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "Nwires = 2\n",
+    "backend = SpinSimulator()\n",
+    "qc_rabi = QuantumCircuit(QuantumRegister(Nwires, \"spin\"))\n",
+    "\n",
+    "all_modes=range(Nwires)\n",
+    "omega_t = Parameter(\"ω\")\n",
+    "\n",
+    "qc_rabi.rlx( omega_t, [0, 1])\n",
+    "\n",
+    "qc_rabi.measure_all()\n",
+    "qc_rabi.draw(output='mpl')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "b05af072-2603-4506-9911-689f5017f02f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "phases = np.linspace(0, 2*np.pi,15)\n",
+    "\n",
+    "rabi_list = [\n",
+    "    qc_rabi.assign_parameters(\n",
+    "        {omega_t: phase},\n",
+    "        inplace=False,\n",
+    "    )\n",
+    "    for phase in phases\n",
+    "]\n",
+    "\n",
+    "job_rabi = backend.run(rabi_list, shots=500)\n",
+    "result_rabi = job_rabi.result()\n",
+    "counts_rabi = job_rabi.result().get_counts()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "034b74d1-81d8-4363-a8db-89af6424f442",
+   "metadata": {},
+   "source": [
+    "next we have to count the same of the two spins that are `1` at the same time"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "7d05af5b-80cc-4490-aa9b-5e50a673dced",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "outcomes_rabi = [result_rabi.get_memory(i) for i in range(len(rabi_list))]\n",
+    "for i, outcome in enumerate(outcomes_rabi):\n",
+    "    for j, run in enumerate(outcome):\n",
+    "        outcomes_rabi[i][j] = np.array(run.split(' '), dtype = int)\n",
+    "outcomes_rabi = np.array(outcomes_rabi)"
+   ]
```

Comment:
```suggestion
   "source": [
    "sum_rabi = []\n",
    "for i, _ in enumerate(phases):\n",
    "    counts_up = 0\n",
    "    for outcome, count in counts_rabi[i].items():\n",
    "        if outcome[0] == \"1\":\n",
    "            counts_up += count\n",
    "        if outcome[-1] == \"1\":\n",
    "            counts_up += count        \n",
    "    sum_rabi.append(counts_up/n_shots)"
   ]
```

I think this way to compute `sum_rabi` is a bit more direct and easier to parse than going through the memory like you're doing. Your method is also fine with me though.